src/worker.config.json

```diff
@@ -1,11 +1,12 @@
 {
-    "description":{
-        "language":"powershell",
-        "extensions":[".ps1", ".psm1"],
-        "defaultExecutablePath":"dotnet",
-        "defaultWorkerPath":"%FUNCTIONS_WORKER_RUNTIME_VERSION%/Microsoft.Azure.Functions.PowerShellWorker.dll",
-        "supportedRuntimeVersions":["7", "7.2"],
+    "description": {
```

Comment:
We should not make any changes to the existing `worker.config.json`, so we can continue to have the existing behavior. However, could you please document the required changes in `worker.config.json` to enable `workerIndexing`? Perhaps the `src/WorkerIndexing/WorkerIndexingHelper.cs` would be a good place for this. Let me know what you think.

Reply:
As per our conversation earlier this morning, two flags are necessary in order for the host to attempt worker indexing - the first is in the worker.config.json, and the second is in the local.settings.json/function app environment variables. Having this string in worker.config.json reports to the host that the worker is compatible with worker indexing, and the one in local.settings.json reports that the app is compatible. 
So, long story short, making this change doesn't harm existing apps.